src/overcooked_ai_py/planning/planners.py

```diff
@@ -1,18 +1,22 @@
+# this denotes whether we are running the clean version of planners
+clean = True
```

Comment:
What does clean version of the planners mean? Elaborate in the comment

Reply:
This was added because of circular import of OvercookedEnv. It has been addressed. Thanks for catching it!